vault/resource_jwt_auth_backend.go

```diff
@@ -106,6 +123,36 @@ func jwtAuthBackendResource() *schema.Resource {
 	}
 }
 
+func jwtCustomizeDiff(d *schema.ResourceDiff, meta interface{}) error {
```

Comment:
This is checking that existence is >= 1. Consider something like https://github.com/hashicorp/vault-plugin-auth-jwt/blob/master/path_config.go#L175 to check that exactly one method is present.